homeassistant/components/overkiz/manifest.json

```diff
@@ -30,5 +30,12 @@
     "pyhumps",
     "pyoverkiz",
     "s3transfer"
-  ]
+  ],
+  "supported_brands": {
+    "cozytouch": "Atlantic Cozytouch",
+    "flexom": "Bouygues Flexom",
+    "hi_kumo": "Hitachi Hi Kumo",
+    "nexity": "Nexity Eugénie",
+    "rexel": "Rexel Energeasy Connect"
```

Comment:
@vlebourl, @tetienne any suggestions for the names? Do we just want to use the brand name, or do we want to add the 'type' name as well? Or just the name of the hub?

I can see a potential conflict around Cozytouch where you have "Atlantic Cozytouch" and "Thermor Cozytouch", if I recall correctly.

Reply:
Well, if we just talk about brand, we haven’t to talk about cozytouch but only Atlantic for instance. So I’m confuse on what to add here. 
And we must also add Somfy, but somfy domain is for sure already taken by the Somfy integration.

Reply:
> And we must also add Somfy, but somfy domain is for sure already taken by the Somfy integration.

Indeed, let's keep the name Overkiz (by Somfy) for now, until the other Somfy integration is removed.